empack_config.yaml

```diff
@@ -0,0 +1,48 @@
+packages:
+  python-dateutil: 
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - regex: '*dateutil-zoneinfo.tar.gz'
+   exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+
+  matplotlib:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - regex: '*dateutil-zoneinfo.tar.gz'
+   exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+      - pattern: "*matplotlibrc"
```

Comment:
We actually need this

```suggestion
```

Reply:
@martinRenou this was not yet ready for review and had a lots of errors, now it should be ready for review